src/__tests__/cacheDirectory.test.ts

```diff
@@ -0,0 +1,176 @@
+import { promises as fs } from 'fs';
+import { v4 as uuid } from 'uuid';
+import { vol } from 'memfs';
+
+import {
+  getDefaultCacheDirectory,
+  writeJsonToPath,
+  symlink,
+} from '../cacheDirectory';
+
+jest.mock('fs'); // applies manual mock which uses memfs
+
+afterEach(() => {
+  jest.clearAllMocks();
```

Comment:
How do you feel about using the option in a jest config: https://jestjs.io/docs/en/configuration#clearmocks-boolean I prefer it because I always want mocks cleared, and it saves from having to add this line in test files.

Reply:
I'm cool with changing that. Will make the tweak.